a2i-code-For-Govstack/sp-service-solutions-api
# [3] – Data app bug fixes and enhancements

## Description

### This PR aims to add the following
  - Adds functionality to check if all the required fields are passed or not; it not, an error is thrown.
  - Adds functionality to check and validate that one social user only votes one time.
  - Answer field now accepts JSON input, helping in single-option and multi-option.
  - Description field is added in form/sleketon to add form description.

 ### How to test
- Setup the database configs in `config/settings.py`
- Make a `.env` file based on `.env.example`
- Migrate the database and start the server.
- Check all the mentioned functionalities.

Signed by Divij Sharma

CC @Sanchay-T
